### PR TITLE
fix saas sparkdesk llm code

### PR DESCRIPTION
**Pull request by rorovic**
1. 增加gpt_url, DOMAIN参数，让用户自定义gpt版本和domain参数
2. 改写CustomSaasAPI获参方式

### CODE REVIEW

File: src/byzerllm/saas/sparkdesk/__init__.py [hunk 1]
```diff
@@ -60,28 +61,29 @@ def create_url(self):
         # 此处打印出建立连接时候的url,参考本demo的时候可取消上方打印的注释，比对相同参数时生成的url与自己代码生成的url是否一致
         return url
 
-class SparkDeskAPI:
+class CustomSaasAPI:
+
+    def __init__(self, infer_params: Dict[str, str]) -> None:
+        self.appid: str = infer_params.get("saas.appid","")
```
Comment (allwefantasy) [inline src/byzerllm/saas/sparkdesk/__init__.py:67]: The parameters in this three lines are required not optional. If the user miss this parameters, we should throw exception.

**Review by allwefantasy — CHANGES_REQUESTED** [review]

File: src/byzerllm/saas/sparkdesk/__init__.py [hunk 2]
```diff
@@ -60,28 +61,29 @@ def create_url(self):
...
+        self.api_key: str = infer_params.get("saas.api_key","")
+        self.api_secret: str = infer_params.get("saas.api_secret","")
+        self.gpt_url: str = infer_params.get("saas.gpt_url","")
```
Comment (allwefantasy) [inline src/byzerllm/saas/sparkdesk/__init__.py:70]: When the user didn't configure git_url/domain , what will happen?

**Review by allwefantasy — CHANGES_REQUESTED** [review]

File: src/byzerllm/saas/sparkdesk/__init__.py (hunk 1 above)
Comment (rorovic) [reply src/byzerllm/saas/sparkdesk/__init__.py:67]: OK, I will add the exception handling

File: src/byzerllm/saas/sparkdesk/__init__.py (hunk 2 above)
Comment (rorovic) [reply src/byzerllm/saas/sparkdesk/__init__.py:70]: I will give domain a default value

**Comment by rorovic** [comment]
1 增加gpt_url, DOMAIN参数，让用户自定义gpt版本和domain参数
2 改写CustomSaasAPI获参方式，增加参数空值异常判断
3 README 增加sparkdesk Python例子和模型说明

**Review by allwefantasy — APPROVED** [review]

**Pull request merged**